README.md

```diff
@@ -38,23 +38,24 @@ Curtesy of [this](http://googleresearch.blogspot.ru/2015/11/computer-respond-to-
 
     git clone git@github.com:nicolas-ivanov/tf_seq2seq_chatbot.git
     cd tf_seq2seq_chatbot
-    bash bin/setup.sh
+    chmod +x setup.sh
```

Comment:
do we really need this line?
seems to me `bash setup.sh` gonna do the job all the same